src/Components/Footer/Footer.tsx

```diff
@@ -0,0 +1,80 @@
+import classNames from 'classnames';
+import { Links } from '../../utils/enum';
+import { Todo } from '../../types/Todo';
+import { deleteTodo } from '../../api/todos';
+
+type Props = {
+  filteredBy: string;
+  todosLengh: number;
```

Comment:
There is a typo in the property name `todosLengh`. Consider renaming it to `todosLength` to avoid confusion.